coderd/workspaceagents.go

```diff
@@ -321,6 +323,7 @@ func (api *API) postWorkspaceAgentStartup(rw http.ResponseWriter, r *http.Reques
 		Version:           req.Version,
 		ExpandedDirectory: req.ExpandedDirectory,
 		Subsystems:        convertWorkspaceAgentSubsystems(req.Subsystems),
+		APIVersion:        AgentAPIVersionREST,
```

Comment:
Shouldn't this be reported by the agent instead of defined in coderd? If the agent is old there's no guarantee it supports this version, or am I missing something?

Reply:
We are _defining_ the current REST API as Version 1.0 of the Agent API.

Version 2.0 will be based on dRPC, and the Agent will report the version it wants when it connects the websocket.  If the agent is connecting via REST, then it must be 1.0.

Reply:
Ah, alright. That makes sense, thanks for clarifying 👍🏻